content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -438,7 +440,14 @@ Here `v1.8.x` means the "latest patch release of the v1.8 branch".
 
 `${ARCH}` can be one of: `amd64`, `arm`, `arm64`, `ppc64le` or `s390x`.
 
-If using `--feature-gates=CoreDNS=true` image `coredns/coredns:1.0.2` is required (instead of the three `k8s-dns-*` images).
+In Kubernetes versions before 1.11 and if using `--feature-gates=CoreDNS=true`, the image `coredns/coredns:1.0.2` is required (instead of the three `k8s-dns-*` images).
```

Comment:
Is this content about older versions needed? @Bradamant3 thought it might not be. If it is, I'd like to do some copy-editing on this. Please let me know. Thanks!

Reply:
it is kind of valid, because the  `k8s-dns-*` images will not be needed post 1.11, by default.
please advice on the changes in mind.

Reply:
@mistyhacks

Reply:
I still don't understand, but it seems that @Bradamant3 had thoughts about this so maybe she'll weigh in tomorrow.